From 626911079bb62d448727f6bcd9b172052a810418 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 14 Nov 2017 21:02:21 +0800
Subject: [PATCH 2802/5242] MLK-16810 ARM64: dts: freescale: imx8mq: set timer
 stop flag in suspend

commit  c689174d4c041e67029b61a025d8db9f7e06bd0b from
https://source.codeaurora.org/external/imx/linux-imx.git

System counter's clock is disabled in suspend, set flag
for arch timer driver's awareness.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 336024a..2486cdf 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -110,6 +110,7 @@
 			     <GIC_PPI 10 (GIC_CPU_MASK_SIMPLE(6) | IRQ_TYPE_LEVEL_LOW)>; /* Hypervisor */
 		clock-frequency = <8333333>;
 		interrupt-parent = <&gic>;
+		arm,no-tick-in-suspend;
 	};
 
 	busfreq { /* BUSFREQ */
-- 
1.7.9.5

